### PR TITLE
Quantize elementwise sub

### PR DESCRIPTION
**Pull request by zuzg**
### PR types
New features

### PR changes
OPs

### Describe
* Quantization of elementwise subtraction
* Slight refactoring of tests for quantized elementwise operators

### CODE REVIEW

**Review by wozna — APPROVED** [review]
LGTM

**Review by sfraczek — APPROVED** [review]
LGTM

File: paddle/fluid/framework/ir/mkldnn/cpu_quantize_pass_tester.cc [hunk 1]
```diff
@@ -565,60 +566,59 @@ ProgramDesc BuildProgramDescElementwise(const std::string elementwise_type,
...
+    : public testing::TestWithParam<std::vector<std::string>> {};
 
-TEST(CpuQuantizePass, elementwise_add_unsigned_and_signed_input) {
-  TestElementwiseUnsignedAndSignedInput("elementwise_add", "ElementwiseAdd");
-}
+TEST_P(TestElementwises, elementwise_basic) { TestElementwise(GetParam()); }
 
-TEST(CpuQuantizePass, elementwise_mul) {
-  TestElementwise("elementwise_mul", "ElementwiseMul");
+TEST_P(TestElementwises, elementwise_output_scale_missing) {
```
Comment (lidanqing-vv) [inline paddle/fluid/framework/ir/mkldnn/cpu_quantize_pass_tester.cc:608]: Hi, should this highest level of name remains `CpuQuantizePass`? now it is changed to `TestElementwises`, I worry there can be conflicts with elementwise op it self tests.

**Review by lidanqing-vv — APPROVED** [review]
LGTM

File: paddle/fluid/framework/ir/mkldnn/cpu_quantize_pass_tester.cc (hunk 1 above)
Comment (lidanqing-vv) [reply paddle/fluid/framework/ir/mkldnn/cpu_quantize_pass_tester.cc:608]: @zuzg Please change it in next PR for post-training elementwise_sub .

**Pull request merged**